apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -589,5 +589,41 @@ public Response dexEthInfo(@Context SecurityContext securityContext) throws NotF
             return Response.ok().build();
         }
     }
+    
+    
+    
+    @GET
+    @Path("/flush")
```

Comment:
maybe logout?

Reply:
no, since we have a reserved global system login/logout. In this situation, it might be either 'flush' (like water in the loo) or 'wipe'. 'Flush' sounds better, as for me.